modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -231,24 +238,51 @@ $ oc get configmap kubevirt-vm-latency-checkup -n kiagnose -o yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kubevirt-vm-latency-checkup
-  namespace: kiagnose
-...
+  name: kubevirt-vm-latency-checkup-config
+  namespace: <target_namespace>
+data:
+  spec.image: registry.redhat.io/container-native-virtualization/vm-network-latency-checkup:v4.12.0
+  spec.timeout: 5m
+  spec.serviceAccountName: vm-latency-checkup-sa
+  spec.param.network_attachment_definition_namespace: <target_namespace>
+  spec.param.network_attachment_definition_name: "bridge-network"
+  spec.param.max_desired_latency_milliseconds: "10"
+  spec.param.sample_duration_seconds: "5"
+  spec.param.source_node: "worker1"
+  spec.param.target_node: "worker2"
   status.succeeded: "true"
   status.failureReason: ""
-  status.result.minLatencyNanoSec: 2000
-  status.result.maxLatencyNanoSec: 3000
-  status.result.avgLatencyNanoSec: 2500
-  status.results.measurementDurationSec: 300
-...
+  status.completionTimestamp: "2022-01-01T09:00:00Z"
+  status.startTimestamp: "2022-01-01T09:00:07Z"
+  status.result.avgLatencyNanoSec: "177000"
+  status.result.maxLatencyNanoSec: "244000" <1>
+  status.result.measurementDurationSec: "5"
+  status.result.minLatencyNanoSec: "135000"
+  status.result.sourceNode: "worker1"
+  status.result.targetNode: "worker2"
 ----
+<1> The maximum measured latency in nanoseconds.
 
-. Delete the framework and checkup resources that you previously created. This includes the job, config map, cluster role, and framework manifest files.
+. Optional: To view the detailed job log in case of checkup failure, use the following command:
 +
-[NOTE]
-====
-Do not delete the framework and cluster role manifest files if you plan to run another checkup.
-====
+[source,terminal]
+----
+$ oc logs job.batch/kubevirt-vm-latency-checkup -n kiagnose
+----
+
+. Delete the job and config map resources that you previously created.
```

Comment:
Since this is two commands, I would try to draw attention to that by adding "...by running the following commands:"